CalculatorLibrary/CalculatorLibrary.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.IO;
+using System.Diagnostics;
+using Newtonsoft.Json;
+
+namespace CalculatorLibrary
+{
+    public class CalculatorProgram
+    {
+        JsonWriter writer;
+        private int _calUsageCount = 0;
+        //private string _histNum;
+        private double _returnSelectHist;
```

Comment:
Is this returning the selected history. Why shorten history, shortening it makes it a bit confusing.